pytorch_blade/bazel_build.py

```diff
@@ -152,6 +152,8 @@ def __init__(self, *args, **kwargs):
         if running_on_ci():
             self.configs += ["--config=ci_build"]
 
+        self.configs += [" --linkopt='-fuse-ld=gold -Xlinker --stub-group-size -Xlinker 10000000'"]
```

Comment:
Nit: We could extend `extra_link_flags` to receive an default argument like `force_gold=False` or whatever, to encapsulate gold default argument into a single place.